packages/clay-css/src/scss/components/_modals.scss

```diff
@@ -66,6 +66,11 @@
 
 	.modal-content {
 		@include box-shadow($modal-content-box-shadow-sm-up);
+
+		.modal-body,
+		.modal-header {
+			padding: 1.5rem;
```

Comment:
@ilzamcmed This will be a more complicated fix unfortunately. I want to leverage https://github.com/liferay/clay/blob/b1cf8c491ef7b9020ca647aa2b69c1e622c43821/packages/clay-css/src/scss/mixins/_grid.scss#L203-L261 and https://github.com/liferay/clay/blob/b1cf8c491ef7b9020ca647aa2b69c1e622c43821/packages/clay-css/src/scss/mixins/_globals.scss#L370-L396.

Let me know if you're up for it or I can send an update to this one.